### PR TITLE
Week 0:Introductory blog post

### PR DESCRIPTION
**Pull request by amitjamadagni**
@i2000s  This is the introductory blog post. I have tested its appearance by hosting the github.io page on localhost. Though some of it needs some tweaks. I guess this would be the structure in updating the progress of the project. Hoping to hear some reviews.

### CODE REVIEW

File: news/_posts/2015-04-29-JuliaQuantum-and-GSoC2015.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Once we have the initial framework for solvers, we would be aiming to parallelize the implementation at various stages i.e., at an abstract level as QuArray is dependent on Arrays and also at various stages in the methods.
+
+As per the schedule of the project we are currently in the Community Bonding Period and here is my understanding of how things are moving and also what we plan to achieve before the coding period :
+
+1. Currently there is a [pull request](https://github.com/JuliaQuantum/QuBase.jl/pull/29) in progress to replace the `Union(QuArray, CTranspose)` type with something more abstract like `AbstractQuArray` and then having subtypes of this and each subtype should have `coefftype` which gives the type of the coefficients, `rawcoeffs` the coefficients themselves, `rawbases` the related bases, and `copy` which gives a copy of the construct. So the type structure is something like
+
+`AbstarctQuArray` --- top-node
```
Comment (garrison) [inline news/_posts/2015-04-29-JuliaQuantum-and-GSoC2015.md]: typo here

File: news/_posts/2015-04-29-JuliaQuantum-and-GSoC2015.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      |
+      | ---  `QuArray`
+      | ---  `CTranspose` (immutable)
+      | ---  `QuDensity` (TBI)
+
+Regarding the promotion rules, this was a bit difficult to understand at first but here is what I could get from the PR
+Suppose we have a `QuArray` and `CTranpose` and we need to operate with an instance from each, then we need a rule to decide which type instance would be returned and hence the promotion rule. The latest being that we default to QuArray if they are two different types.
```
Comment (garrison) [inline news/_posts/2015-04-29-JuliaQuantum-and-GSoC2015.md]: Tranpose -> Transpose

**Comment by acroy** [comment]
I like it. In the first 3 paragraphs you can replace "would" by "will" :-)

**Comment by garrison** [comment]
Looks like a good introduction to me, but I was only able to skim it.  I may try reading it again tomorrow if I get a chance.

**Comment by jrevels** [comment]
Looks good!

**Comment by i2000s** [comment]
@amitjamadagni Seems the repo for this project has been initialized. Should you be able to add a link to the repo on this blog post? https://github.com/JuliaQuantum/QuDynamics.jl

We may want the final confirm from @acroy -- and actually he is able to merge this post to the master branch. @acroy and a few others should be able to forward this post through our Google+ and Twitter networks afterwards. Thank everyone here!

**Comment by amitjamadagni** [comment]
@i2000s I have added the link of the repo to the blog. @acroy I guess this is ready to go and also I hope that Gemfile.lock should not be a problem. But please do let me know. Thanks once again @i2000s for all the help :)

**Comment by acroy** [comment]
I have no idea about `Gemfile.lock` - do we use that at all? I guess we have to merge this to see the impact ...

**Comment by i2000s** [comment]
The `Gemfile.lock` file is generated while you are running the localhost. It doesn't matter to the website on GitHub, and that file could be deleted in the future. The only issue I can tell is the title seems a little bit long so that the date of writing has been pushed away to the next line on the `news` page. But it is not crucial.

Hopefully plenty of peer-reviewers can be drawn to the correct repo.

**Comment by acroy** [comment]
@i2000s : Thanks for the info. We should probably remove the file then.

> Hopefully plenty of peer-reviewers can be drawn to the correct repo.

On Monday (or so) we will send an email to julia-users and also write something via Twitter/Google+. Once we have some content we should link the new repo on our "projects" site.

**Comment by i2000s** [comment]
@acroy All sounds good. I will be sharing the progress among my personal network using the *Share* buttons on the right-hand-side of the webpage.

**Pull request merged**